sw/device/rom_exts/manifest.hjson

```diff
@@ -48,6 +56,7 @@
             ''',
       type: "field",
       size: 64,
+      alignment: 64,
```

Comment:
I feel like it would be useful to mention that the alignment of 64 is particularly useful for other non-ibec platforms such as RV64 and other  non-RISC-V 64 bit platforms. But I don't feel like this would be the best place for it.

Reply:
I'm happy to leave this as "`manifest.md` requires 64-bit alignment, so we're just copying that information into this machine-readable description", and not add a comment here. 

Yes, the 64-bit alignment doesn't make a huge difference on Ibex, but it does feel nicer not to under-align scalar fields.